docs/src/search.md

```diff
@@ -107,6 +111,33 @@ an ANN search means that using an index often involves a trade-off between recal
 See the [IVF_PQ index](./concepts/index_ivfpq.md) for a deeper description of how `IVF_PQ`
 indexes work in LanceDB.
 
+## Binary vector
+
+LanceDB supports binary vectors as a data type, and has the ability to search binary vectors with hamming distance. The binary vectors are stored as uint8 arrays:
```

Comment:
Is each bit in the uint8 array a value? or each byte? Could you describe that format a little more?

Reply:
added